utils/tests/test_appliance_cli.py

```diff
@@ -0,0 +1,102 @@
+from fixtures.pytest_store import store
+from utils.version import get_stream
+from utils.sprout import SproutClient
+from utils.appliance import current_appliance
+from utils.conf import cfme_data, credentials
+from utils.log import logger
+import pytest
+
+
+@pytest.yield_fixture(scope="module")
+def sprout_provisioned_appliance():
+    sp = SproutClient.from_config()
+    version = current_appliance.version.vstring
+    stream = get_stream(current_appliance.version)
+    apps, pool_id = sp.provision_appliances(
+        count=1, preconfigured=False, version=version, stream=stream)
+
+    yield apps[0]
+
+    sp.destroy_pool(pool_id)
+
+
+@pytest.yield_fixture(scope="module")
+def sprout_provisioned_appliance2():
```

Comment:
In fact, we don't even really need sprout in there....how about just ```fqdn_appliance``` and ```appliance``` for the one before